lib/jobs/user_match_runner.ts

```diff
@@ -1,7 +1,7 @@
 import { runBackfills, runUpdates } from '../etl/user_match_scrape'
 import { createEtlJob, updateEtlJob } from '../model/etl_jobs'
 import { argv } from 'yargs'
-import * as md5 from 'md5'
```

Comment:
@bbleaptrot When I was doing some TypeScript migration stuff, I accidentally broke the ETL jobs. This should fix them.